browser/about_flags.cc

```diff
@@ -380,6 +383,17 @@ constexpr char kFileSystemAccessAPIDescription[] =
 #define BRAVE_TRANSLATE_GO_FEATURE_ENTRIES
 #endif  // BUILDFLAG(ENABLE_BRAVE_TRANSLATE_GO)
 
+#if !defined(OS_ANDROID)
+#define BRAVE_SHIELDS_V2_FEATURE_ENTRIES                         \
+    {kBraveShieldsV2FeatureInternalName,                        \
```

Comment:
`brave-shields-v2` can be used here instead of this because this name is not referenced from other places.